toolkit/tools/imagecustomizerapi/service.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+import (
+	"fmt"
+)
+
+type Services struct {
+	Enable  []Service `yaml:"Enable"`
```

Comment:
Should that be string of services names instead?

Reply:
Or is the goal we can support path/additional flags in the future?

Reply:
I thought it is more extensible to wrap the name inside a Service struct, and it potentially allow for more attributes or methods related to a service in the future without breaking changes. If you believe that this added complexity won't provide value in our context, I'm open to simplifying it to []String